_posts/2024-07-22-developer-update.md

```diff
@@ -0,0 +1,115 @@
+
+Security integrations often require infrastructure to develop and test against, which the core GeoServer team does not have access to for automated tests. While the initial priority will be basic OAuth2 and OIDC parties interested in maintaining support for Google, Azure, GeoNode, etc... are welcome to participate.
+
+Interested Parties:
+
+- Andreas (ITS Digital Solutions GmbH) offered to work on this activity and has indicated they are now ready to start
+- GeoCat is in position to do this work after performing the same upgrade for the GeoNetwork codebase. GeoCat wishes to have an OIDC plugin brought up to full extension status, if financing is available.
+
+Activity: not started
+
+* [[GEOS-11272]](https://osgeo-org.atlassian.net/browse/GEOS-11272) spring-security-oauth replacement, with spring-security 5.8
+
+### ImageN / JAI Replacement
+
+The image processing library used by GeoServer has been donated to the open source community under the name [ImageN](https://projects.eclipse.org/projects/technology.imagen).
+
+We would like to see organisations that depend on GeoServer for earth observation and imagery to step forward with funding for this activity.
+
+Interested Parties: 
+
+* Jody (GeoCat) is available to support this activity, or take lead if funding is available.
+* Andrea (GeoSolutions) has had a deep dive into the implications for the JAI-EXT project outlining a roadmap for project integration
+
+Activities:
+
+* The immediate goal is to add test cases to this codebase and make a version 1.0 release
+
+## Other Development activities
+
+In addition to the key roadmap activities a few other development ideas have been shared:
+
+* OGCAPI Web Services: Gabe (camptocamp) and Jody (GeoCat) are interested in setting up a code sprint for September on this activity.
+  
+  If there is enough interest we would like to arrange a cost-recovery code sprint model as was done for last years API change. Ideally we would like to see a mix of developers, and module lead Andrea (GeoSolutions), attend if sufficient funding is available.
+
+* S57 / S100: Jody (GeoCat) asked on the user list for any parties interested in the S57 and S100 marine formats. Such an activity may result in direct GeoTools implementation, or revitalization of the OGR datastore as appropriate.
+
+We also have a few requests for assistance:
+
+* Refactor inline JavaScript in the OGC API modules
+  
+  In order to enable a more restricted ``Content-Security-Policy`` Steve Ikeoka has carefully removed all inline JavaScript from the OGCAPI community modules. Before pull-request [geoserver#7587](https://github.com/geoserver/geoserver/pull/7587) can be merged we need a volunteer to do A/B testing on each page, to double check we have not lost anything.
```

Comment:
Both OGC API and these can be roadmap, but are also off-topic compared to the lack of funding for Jakarta EE... aren't you going to just confuse the reader?

Reply:
Yeah I can remove this section, short and sweet.